### ISSUE TITLE
Notice: Use of undefined constant LC_MESSAGES - assumed 'LC_MESSAGES'

### ISSUE THREAD

**Issue by ve3** [oscarotero/Gettext#89]
my code:
```
<?php 

require __DIR__.'/vendor/autoload.php';


setlocale(LC_ALL, 'en-US');

$t = new \Gettext\GettextTranslator();

//Set the language and load the domain
$t->setLanguage('gl');
$t->loadDomain('messages', 'Locale');
```

And there are errors following...

> Deprecated: setlocale(): Passing locale category name as string is deprecated. Use the LC_* -constants instead

> Warning: setlocale(): Invalid locale category name LC_MESSAGES, must be one of LC_ALL, LC_COLLATE, LC_CTYPE, LC_MONETARY, LC_NUMERIC, or LC_TIME

**Comment by oscarotero** [comment]
It seems that `LC_MESSAGES` is not available. According with [the documentation](http://php.net/manual/en/function.setlocale.php), this constant is available only if PHP was compiled with *libintl*. So `Translator::setLanguage()` should check whether the constant exists before use them, and if not, use LC_ALL instead.
Thanks for the report, I'll fix this ASAP.

**Comment by oscarotero** [comment]
Released [v3.5.7](https://github.com/oscarotero/Gettext/releases/tag/v3.5.7)
